src/renderer/main.js

```diff
@@ -1,11 +1,12 @@
 // import the styles
```

Comment:
This comment can probably be removed as it doesn't seem to be relevant to anything happening near the top of this file anymore.
```suggestion
```